obby-islands/README.md

```diff
@@ -0,0 +1,22 @@
+# obby-islands
+
+This is a Hiber3D HDK project created using [create-hdk](https://github.com/hiberworld/codekit/packages/create-hdk)
```

Comment:
This link should go to npmjs, not our private repo

Reply:
Fixed!